specs/ClientCertificate.md

```diff
@@ -559,14 +559,18 @@ namespace Microsoft. Web. WebView2. Core
         CoreWebView2ClientCertificateKind Kind { get; };
 
         String ToPemEncoding();
-        /// Converts this to a System.Security.Cryptography.X509Certificates.X509Certificate2.
-        /// This is only for the .NET API, not the WinRT API.
+        // Converts this to a System.Security.Cryptography.X509Certificates.X509Certificate2.
```

Comment:
This part is good to include as documnetation. The following line about this only being for .NET or WinRT should be // since its really a comment for API reviewers and not our documentation and so that // comment is good
```suggestion
        /// Converts this to a System.Security.Cryptography.X509Certificates.X509Certificate2.
```